docker-storage-setup.sh

```diff
@@ -895,8 +883,8 @@ setup_storage() {
   # Set up lvm thin pool LV
   if [ "$STORAGE_DRIVER" == "devicemapper" ]; then
     setup_lvm_thin_pool
-  elif [[ "$STORAGE_DRIVER" == "overlay" || "$STORAGE_DRIVER" == "overlay2" ]];then
-    setup_overlay
+  else
```

Comment:
I liked that original elif condition. That is we write config file only if graph driver is overlay or overlay2. Now we are putting blindly in else loop so it will be true for any value other than devicemapper. While it is true as of now, it might not be true in future when we support more graph drivers.